core/src/edu/cornell/gdiac/rabbeat/ObjectController.java

```diff
@@ -498,12 +503,21 @@ public void populateObjects(Vector2 scale) {
                             float y = platform.getFloat("y");
                             Vector2 dim = new Vector2(platform.getFloat("width"), platform.getFloat("height"));
                             String align = "";
-                            for (JsonValue prop : platform.get("properties")) {
```

Comment:
You deleted this line of code that is crucial for beatList to work

Reply:
I didn't, I replaced it with a if statement that checked to make sure properties wasn't null first.

Reply:
![image](https://github.com/JacobSeto/RabBeat/assets/93808580/b54518a4-8c5d-447c-a58c-0d5eba58ca4d)